packages/config/src/tvl/getTvlAmountsConfig.ts

```diff
@@ -72,15 +75,39 @@ export function getTvlAmountsConfig(
             coingeckoId: token.coingeckoId,
             project: projectId,
             source: token.source,
+            dataSource: 'coingecko',
+            includeInTotal: true,
+            decimals: token.decimals,
+            symbol: token.symbol,
+            isAssociated,
+            category: token.category,
+          })
+          break
+        case 'preminted':
+          assert(token.address, 'Token address is required for preminted')
+          assert(token.escrow, 'Escrows are required for preminted')
+
+          entries.push({
+            type: 'preminted',
+            address: token.address,
+            escrowAddress: token.escrow,
+            chain: chainConverter.toName(token.chainId),
+            sinceTimestamp,
+            untilTimestamp: token.untilTimestamp,
+            coingeckoId: token.coingeckoId,
+            project: projectId,
+            dataSource: `${chainConverter.toName(token.chainId)}_preminted_${
+              token.address
+            }`,
+            source: token.source,
+            // TODO
```

Comment:
?

Reply:
good catch, resolved, we will include only tokens on ethereum